src/app.py

```diff
@@ -71,29 +73,19 @@ def lambda_handler(event, context):
     :param context: AWS Lambda uses this parameter to provide your handler the runtime information of the Lambda function that is executing.
     :return: None
     """
-    garmin_s3_bucket = os.environ['GARMIN_S3_BUCKET']
-
-    garmin_user = get_secret_value(os.environ['GARMIN_USERNAME'])
-    garmin_pass = get_secret_value(os.environ['GARMIN_PASSWORD'])
-
-    garth_client = Client()
-    garth_client.login(garmin_user, garmin_pass)
-
-    try:
-        client = boto3.client('s3')
-
-        sleep_json_data = sleep_data(garth_client=garth_client)
-        client.put_object(
-            Bucket=garmin_s3_bucket,
-            Key=f'sleep/{datetime.date.today().isoformat()}/{uuid.uuid4()}.json',
-            Body=json.dumps(sleep_json_data)
-        )
-
-        stress_json_data = stress_data(garth_client=garth_client)
-        client.put_object(
-            Bucket=garmin_s3_bucket,
-            Key=f'stress/{datetime.date.today().isoformat()}/{uuid.uuid4()}.json',
-            Body=json.dumps(stress_json_data)
-        )
-    except Exception as e:
-        print(e)
+
+    client = boto3.client('s3')
+
+    sleep_json_data = sleep_data(garth_client=garth_client)
+    stress_json_data = stress_data(garth_client=garth_client)
```

Comment:
Moved these _risky_ statements before the S3 puts, so that you are less likely to end up with a situation where the sleep data is PUT to the bucket while the stress data fails.